web/views/analyses.py

```diff
@@ -351,7 +324,7 @@ def get_file(self, id, filehash):
             for filepath in analysis['generated_files'][file_type]:
                 filepath = filepath.encode('utf-8')
                 if filehash == md5(filepath).hexdigest():
-                    return file_download(filehash)
+                    return file_download(filepath)
```

Comment:
Not sure what this change is doing here. I don't think this is related to preloading modules so it should be removed.

Reply:
No, this was an unrelated bugfix. Will remove it from this PR.